app/Http/Controllers/TestController.php

```diff
@@ -0,0 +1,11 @@
+<?php
+namespace App\Http\Controllers;
+
+class TestController extends Controller{
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition

<!-- Comment by [SideCI](https://sideci.com) -->